boards/imix/src/main.rs

```diff
@@ -395,19 +406,76 @@ pub unsafe fn reset_handler() {
-        capsules::mac::MacDevice::new(rf233),
-        0);
+        capsules::mac::MacDevice::new(rf233));
     let radio_capsule = static_init!(
-        capsules::radio::RadioDriver<'static,
-                                     capsules::mac::MacDevice<'static, RF233Device>>,
-        capsules::radio::RadioDriver::new(radio_mac),
-        0);
+        capsules::radio::RadioDriver<'static>,
+        capsules::radio::RadioDriver::new(radio_mac as &'static Mac));
     radio_capsule.config_buffer(&mut RADIO_BUF);
     radio_mac.set_transmit_client(radio_capsule);
     radio_mac.set_receive_client(radio_capsule);
     rf233.set_transmit_client(radio_mac);
     rf233.set_receive_client(radio_mac, &mut RF233_RX_BUF);
     rf233.set_config_client(radio_mac);
+    radio_mac.set_pan(0xABCD);
+    radio_mac.set_address(0x1008);
+
+    let dummy_ctx_store = static_init!(
+        lowpan_frag_dummy::DummyStore,
+        lowpan_frag_dummy::DummyStore::new(capsules::net::lowpan::Context {
+            prefix: DEFAULT_CTX_PREFIX,
+            prefix_len: DEFAULT_CTX_PREFIX_LEN as u8,
+            id: 0,
+            compress: false,
+        })
+    );
+
+    let tx_state = static_init!(
+        capsules::net::lowpan_fragment::TxState<'static>,
+        capsules::net::lowpan_fragment::TxState::new()
+    );
+
+    let rx_state = static_init!(
+        capsules::net::lowpan_fragment::RxState<'static>,
+        capsules::net::lowpan_fragment::RxState::new(&mut RX_STATE_BUF)
+    );
+
+    let frag_state_alarm = static_init!(
+        VirtualMuxAlarm<'static, sam4l::ast::Ast>,
+        VirtualMuxAlarm::new(mux_alarm)
+    );
+
+    let frag_dummy_alarm = static_init!(
+        VirtualMuxAlarm<'static, sam4l::ast::Ast>,
+        VirtualMuxAlarm::new(mux_alarm)
+    );
+
+    let frag_state = static_init!(
+        capsules::net::lowpan_fragment::FragState<'static,
+            VirtualMuxAlarm<'static, sam4l::ast::Ast>>,
+        capsules::net::lowpan_fragment::FragState::new(
+            radio_mac as &'static Mac,
+            dummy_ctx_store as &'static capsules::net::lowpan::ContextStore,
+            &mut RADIO_BUF_TMP,
+            frag_state_alarm)
+    );
+    frag_state.add_rx_state(rx_state);
+    radio_mac.set_transmit_client(frag_state);
+    radio_mac.set_receive_client(frag_state);
+
+    let lowpan_frag_dummy = static_init!(
+        lowpan_frag_dummy::LowpanTest<'static,
+            VirtualMuxAlarm<'static, sam4l::ast::Ast>>,
+            lowpan_frag_dummy::LowpanTest::new(radio_mac as &'static Mac,
+                                               frag_state,
+                                               tx_state,
+                                               frag_dummy_alarm)
+    );
+
+    frag_state.set_receive_client(lowpan_frag_dummy);
+    tx_state.set_transmit_client(lowpan_frag_dummy);
+    frag_state_alarm.set_client(frag_state);
+    frag_dummy_alarm.set_client(lowpan_frag_dummy);
+    frag_state.schedule_next_timer();
```

Comment:
OK, so either we should have not include this capsule in `imix` by default, or it should be called something other than `*_dummy`.